Exercise02-Basics/Nenad-Gvozdenovic/Basics 03 – Operations, Operators, Precedence/basics03-016.js

```diff
@@ -0,0 +1,12 @@
+/*
+BASICS03-016: A bus leaves from point A to point B with speed of 80 km/h. At the same time, 
+a car leaves from point B to point A with speed of x km/h. The distance between point A 
+and point B is S kilometers. After how many minutes, the bus, and the car will meet? 
+Print the result on the console.
+*/
+
+const speed = 
+let AB = 80 //kmh;
+let BA = x //kmh;
+let distance = s //km;
+let x = (AB - BA)*s
```

Comment:
The speed variable doesn't have any value and sould be written in capital letters(it's constant)